Makefile

```
@@ -76,10 +76,14 @@ image-build: $(foreach i,$(IMAGE_TARGET_LIST),image/$(i)) ## Build all images.
 
 # Build an image.
 IMAGE_REPO ?= quay.io/operator-framework
-image/%: build/%
-	rm -rf build/_image && mkdir -p build/_image && cp build/$* build/_image
-	docker build -t $(IMAGE_REPO)/$*:dev -f ./images/$*/Dockerfile build/_image
-	@rm -rf build/_image
+image/%: BUILD_DIR = build/_image
+# Images run on the linux kernel, so binaries must always target linux.
+image/%: export GOOS = linux
+image/%: image-clean build/%
+	docker build -t $(IMAGE_REPO)/$*:dev -f ./images/$*/Dockerfile $(BUILD_DIR)
+
+image-clean:
+	rm -rf $(BUILD_DIR)
```

Running `image-clean` directly is the same as `rm -rf build`, right? If so, that's a bit confusing.

Can we run `rm -rf $(BUILD_DIR)` after the docker call?

```suggestion
	rm -rf $(BUILD_DIR)
```